### PR TITLE
Documentcardactivity persona

### PR DESCRIPTION
**Pull request by gitjain**
#### Pull request checklist

- [x] Addresses an existing issue: #1332 
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardActivity.tsx [hunk 1]
```diff
@@ -35,14 +35,16 @@ export class DocumentCardActivity extends React.Component<IDocumentCardActivityP
 
     return (
       <div className={ css('ms-DocumentCardActivity-avatar', styles.avatar) }>
-        { person.initials && (
-          <div className={ css('ms-Persona-initials', styles.avatarInitials) } role='presentation'>
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardActivity.tsx]: If we aren't using the avatarInitials, can it be removed from the sass file? Also, does the resulting html use the class ms-Persona-initials? Just want to make sure anyone who is relying on that class won't see this break.

**Comment by micahgodbolt** [comment]
@mikewheaton you had some opinions about the original issue. Is this PR what you were expecting?

**Comment by mikewheaton** [comment]
@micahgodbolt Yes, I think this should do the trick.

File: packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardActivity.tsx (hunk 1 above)
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardActivity.tsx]: I believe ms-Persona-initials are already defined in Persona

**Comment by dzearing** [comment]
Awesome work @gitjain :)

**Pull request merged**